modules/apps/accessibility/accessibility-menu-web/src/main/resources/META-INF/resources/js/AccessibilityMenu.tsx

```diff
@@ -211,25 +214,31 @@ const AccessibilityMenu = (props: Props) => {
 
 						<ul className="list-unstyled mb-0">
 							{(Object.keys(settings) as Array<KEYS>).map(
-								(key, index, array) => (
-									<AccessibilitySetting
-										className={
-											index + 1 < array.length
-												? 'mb-3'
-												: ''
-										}
-										disabled={isSettingsDisabled}
-										key={settings[key].key}
-										label={settings[key].label}
-										onChange={(value) =>
-											handleAccessiblitySettingChange(
-												value,
-												settings[key]
-											)
-										}
-										value={settings[key].value}
-									/>
-								)
+								(key, index, array) => {
+									return (
```

Comment:
You can do immediate return here without having to declare the explicitly.

Reply:
I put some `console.log` inside the `map` function ant didn't change back. 
Thanks for the heads up!

Reply:
Fixed